po/pt_BR.po

```diff
@@ -0,0 +1,189 @@
+msgstr "Iniciar/parar gravação"
+
+#: src/MainWindow.vala:169
+msgid "Light background"
+msgstr "Modo claro"
+
+#: src/MainWindow.vala:170
+msgid "Dark background"
+msgstr "Modo escuro"
+
+#: src/MainWindow.vala:174 data/com.github.mohelm97.screenrecorder.desktop.in:3
+#: data/com.github.mohelm97.screenrecorder.appdata.xml.in:7
+msgid "Screen Recorder"
+msgstr "Gravação de Tela"
+
+#: src/SaveDialog.vala:73
+msgid "Save record as…"
+msgstr "Salvar gravação como…"
+
+#: src/SaveDialog.vala:77
+msgid "Name:"
+msgstr "Nome:"
+
+#: src/SaveDialog.vala:93
+msgid "Folder:"
+msgstr "Pasta:"
+
+#: src/SaveDialog.vala:103
+msgid "Select Screen Records Folder…"
+msgstr "Selecione a pasta para gravações de tela..."
+
+#: src/SaveDialog.vala:124
+msgid "Cancel"
+msgstr "Cancelar"
+
+#: src/SaveDialog.vala:126
+msgid "Save Original"
+msgstr "Salvar original"
+
+#: src/SaveDialog.vala:128
+msgid "Save"
+msgstr "Salvar"
+
+#: src/SaveDialog.vala:171
+msgid "Rendering went awesome"
+msgstr "Renderizado com sucesso"
+
+#: src/SaveDialog.vala:172
+msgid "Click here to open the records folder"
+msgstr "Clique aqui para abrir a pasta de gravações"
+
+#: src/SaveDialog.vala:195
+#, c-format
+msgid "Screen record from %s"
+msgstr "Gravação de tela de %s"
+
+#: src/Widgets/VideoPlayer.vala:100
+msgid "Toggle playing"
+msgstr "Iniciar/parar gravação"
+
+#: data/com.github.mohelm97.screenrecorder.desktop.in:4
+msgid "Simple tool to record the screen using ffmpeg"
+msgstr "Ferramenta simples para gravação de tela usando o ffmpeg"
+
+#: data/com.github.mohelm97.screenrecorder.desktop.in:6
+msgid "com.github.mohelm97.screenrecorder"
+msgstr "com.github.mohelm97.screenrecorder"
+
+#: data/com.github.mohelm97.screenrecorder.desktop.in:10
+msgid "Snapshot;Screenshot;Screen;Record;Recording;"
+msgstr "Retrato;Captura de tela;Tela;Registro;Gravação;"
+
+#: data/com.github.mohelm97.screenrecorder.appdata.xml.in:8
+msgid "A simple screen recording app for daily use."
+msgstr "Um aplicativo simples de gravação de tela para usar no dia a dia."
+
+#: data/com.github.mohelm97.screenrecorder.appdata.xml.in:10
+msgid "A screen recording app using FFmpeg, with a small set of useful features!"
+msgstr "Um aplicativo de gravação de tela que usa o FFmpeg e com um pequeno conjunto recursos úteis!"
+
+#: data/com.github.mohelm97.screenrecorder.appdata.xml.in:11
+msgid "Features include:"
+msgstr "Os recursos são:"
+
+#: data/com.github.mohelm97.screenrecorder.appdata.xml.in:13
+msgid "gif recording"
+msgstr "Gravação de GIF"
+
+#: data/com.github.mohelm97.screenrecorder.appdata.xml.in:14
+msgid "Recording sound from speakers"
+msgstr "Gravação de sons das caixas de som"
+
+#: data/com.github.mohelm97.screenrecorder.appdata.xml.in:15
+msgid "Recording sound from microphone"
+msgstr "Gravação de sons do microfone"
+
+#: data/com.github.mohelm97.screenrecorder.appdata.xml.in:16
+msgid "Available format: MP4, MOV and GIF"
+msgstr "Formatos para salvamento: MP4, MOV e GIF"
```

Comment:
I  think `Formatos disponíveis` would fit better here. :thinking:

Reply:
Dear @diegotoral, this line appears on AppCenter page:

![Captura de tela de 2020-06-27 00 31 08@2x](https://user-images.githubusercontent.com/2041967/85913749-c5276380-b80d-11ea-8d85-4e7b25915067.png)

As a localization task (and not only direct translation), I think the list of features that appears on AppCenter page is better localized as "Formatos para salvamento". In Portuguese, when we use simply "Formatos disponíveis" the setence doesn't match with the other features.

In fact, thinking further about the localization I would like to suggest a new translation. Take a look at this:

> Características incluem:
> (...)
> - Gravação de sons do microfone
> - **Armazenamento nos formatos MP4, MOV e GIF**

I don't know if my way of thinking if clear for you. Please tell me what you think.